homeassistant/components/velbus/__init__.py

```diff
@@ -215,3 +114,9 @@ async def async_migrate_entry(hass: HomeAssistant, config_entry: ConfigEntry) ->
 
     _LOGGER.debug("Migration to version %s successful", config_entry.version)
     return True
+
+
+async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
+    """Set up the actions for the Velbus component."""
+    setup_services(hass)
+    return True
```

Comment:
Let's move this above `async_setup_entry`

Reply:
done